yarn.lock

```diff
@@ -2592,7 +2592,7 @@
 
 "@testing-library/user-event@^12":
   version "12.8.3"
-  resolved "https://registry.npmjs.org/@testing-library/user-event/-/user-event-12.8.3.tgz#1aa3ed4b9f79340a1e1836bc7f57c501e838704a"
+  resolved "https://registry.yarnpkg.com/@testing-library/user-event/-/user-event-12.8.3.tgz#1aa3ed4b9f79340a1e1836bc7f57c501e838704a"
```

Comment:
I don't think it's necessary...

Reply:
This wasn't something that was done manually. I'm guessing maybe yarn did this when I reverted from v14 back to v12.